apps/server/src/services/integration-service/integrationUtils.ts

```diff
@@ -1,34 +1,62 @@
 // any value inside double curly braces {{val}}
+import { formatDisplay } from 'ontime-utils';
+
 const placeholderRegex = /{{(.*?)}}/g;
 
-/**
- * Parses a templated string
- */
-export function parseTemplate(template: string, state: object): string {
-  let parsedTemplate = template;
-  let match;
-  while ((match = placeholderRegex.exec(template)) !== null) {
-    const variableName = match[1];
-    if (Object.hasOwn(state, variableName)) {
-      parsedTemplate = parsedTemplate.replace(match[0], state[variableName]);
-    }
-  }
+function formatDisplayFromString(value: string, hideZero = false): string {
+  let valueInNumber = null;
 
-  return parsedTemplate;
+  if (value !== 'null') {
```

Comment:
this if block feels redundant and makes the code less readable... you should probably just flip the check and return nulll if there is no value instead of complicating this :)